nncf/quantization/layers.py

```diff
@@ -252,7 +252,15 @@ def run_export_quantization(self, x: torch.Tensor):
             level_high = self.level_high
             levels = self.levels
             if self.is_saturation_fix:
-                x = torch.max(torch.min(x, input_high), input_low)
+                if x.size()[0] > 1:
+                    for i, channel in enumerate(x):
+                        try:
+                            x[i] = torch.clamp(channel, min=input_low[i], max=input_high[i])
+                        except TypeError:
```

Comment:
No exception handling should be done here. Either implement explicit logic deciding whether .item() should be taken, or modify the external code so that is guaranteed that .item() should not be taken.

Reply:
Done

Reply:
Done